modules/admin/proc-rbac-config-conditional-policy-file.adoc

```diff
@@ -0,0 +1,166 @@
+[id='proc-rbac-config-conditional-policy-file_{context}']
+= Configuring conditional policies defined in an external file
+
+You can configure and manage conditional policies that are defined in an external file. To define conditional policies, you can directly edit the configuration files and pass them to {product-short}, instead of using the {product-short} web UI or API. You can configure {product-short} to use these files instead of the default files.
+
+.Prerequisites
+* You are logged in to your {ocp-short} account using the {ocp-short} web console.
+* You have defined conditional policies in a YAML file, which includes role references, permission mappings, and conditions.
```

Comment:
I perceive it more as a first step.

Reply:
It seems that having both a YAML file and a CSV file is a prerequisite, as they are the external files we need to prepare. What do you think?

Reply:
I think that the prerequisite should be the CSV file with the intended roles for the conditions in the YAML file. This makes most sense to me since the doc then moves on to creating the ConfigMap with the YAML content (which can be viewed as the first step as mentioned by @themr0c).

Reply:
Done